acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -1,3 +1,5 @@
+import {createTag} from "../../scripts/miloUtils.js";
```

Comment:
this will make an extra network request at a critical time and slow down our performance